### ISSUE TITLE
Log Cleaner does not read collection setting

### ISSUE THREAD

**Issue by xavi7th** [danielme85/laravel-log-to-db#28]
Hello @danielme85 

I observed a bug. In my logging configuration, I set the collection instance to be something different from the default log. However when I try to prune using the log:delete command, it keeps failing with an error.
```
   Illuminate\Database\QueryException 
  SQLSTATE[42S02]: Base table or view not found: 1146 Table 'test.log' doesn't exist (SQL: select count(*) as aggregate from `log`)
```

I tried to look in the source and noticed that in LogCleanerUpper.php, both calls to the `LogToDb::model() ` method seem to be outdated. 

The LogToDb model has a signature of 
```public static function model(string $channel = null, string $connection = 'default', string $collection = null)```, but both calls 
are done this way ```LogToDB::model($connection, $collection)```.

This makes the collection perpetually null and therefore falls back to default.

Doing this fixes the error for me ```LogToDB::model(null, $connection, $collection)``` but this may not be what you'd prefer.

Kindly look into it and thanks for the great work.

**Comment by danielme85** [comment]
Hi @xavi7th,
thank you for another bug report, I appreciate the help!
I think this bug has been in there since I added that command. I should improve the testing and add this scenario, sorry about that!
Feel free to submit a PR if you want to, but please don't feel any pressure to do so. I think your suggestions should work. It's a little confusing: either 'channel' and/or ('connection', 'collection') would work. So in this command, we have all 3 vars to choose from: but like you said $connection gets assigned as $channel, and that's no good!

You can do: `LogToDB::model(null, $connection, $collection)` like you suggested.

If you don't want to do a PR that's totally fine, I just think you should have the credit for the fix since you found it and provided a workable solution 👍

**Comment by danielme85** [comment]
I think this should be fixed on master branch now: e4def4bfba5e2910cd470334d32699d186fe89cb
Thank you for all the help @xavi7th, I appreciate it! 

If you could give dev-master a try on your setup they would be great.
I'll probably tag another minor release soon either way since this was obviously a bug.

PS: your solution would have worked just fine btw, I just simplified it a little by using log channel names instead since the logging channels in the logging config is a uniquely indexed array anyways.

**Comment by danielme85** [comment]
This fix should be included in v2.4.1 now.
